tests/components/http/Response.js

```diff
@@ -58,6 +58,7 @@ router.post(endpoint, async (request, response) => {
 
 // Bind router to webserver
 const { TEST_SERVER } = require('../Server.js');
+const { test_response_send_status } = require('./scenarios/response_send_status.js');
```

Comment:
nitpick: the require() statement could be on top of the file like the rest of the require